guides/doc-Managing_Hosts/topics/Global_Registration.adoc

```diff
@@ -0,0 +1,185 @@
+[id="Global_Registration"]
+= Registering Host with Global Registration
+
+With Global Registration feature hosts can be registered to the Foreman with
+just one command.
+Main power of this feature came from utilization of provisioning templates,
+which gives user complete control over the process of the host registration.
+
+
+.Prerequisites
+- Foreman user must have `create_hosts` permission
+- Host OS must have assigned Registration template.
+  Go to the template detail > Associations & select the OS
+  Then on OS detail (Hosts > Operating Systems) set default registration template
+- Root privileges on the host that's going to be registered.
+
+
+.Procedure
+. Login to {ProjectName}
+. Go to Hosts > All Hosts > Register Host
+. Set required parameters and generate the command
```

Comment:
Are there parameters that are always required?

Reply:
The only required parameter is `activation_key`, but only for Red Hat family OS.
I see that I forgot to mention it in the list of parameters.